packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -266,10 +266,28 @@ addGraphQLResolvers({
           rating: 0
         }
       })
-    }
+    },
+    async UsersReadPostsOfTargetUser(root: void, { userId, targetUserId, limit = 20 }: { userId: string, targetUserId: string, limit: number }, context: ResolverContext) {
+      const { currentUser, repos } = context
+      if (!currentUser) {
+        throw new Error('Must be logged in to view read posts of target user')
+      }
+
+      const posts = await repos.posts.getUsersReadPostsOfTargetUser(userId, targetUserId, limit)
+
+      return posts
```

Comment:
These need to be run through an `accessFilterMultiple`, since in principle there could be posts that have been drafted since they were read (or should be inaccessible for other reasons)

Reply:
like this?

Reply:
You can just return the output of `accessFilterMultiple` directly without reassigning `posts`